### PR TITLE
chore: New credential UI 

### PR DESCRIPTION
**Pull request by sruddy**

### CODE REVIEW

File: src/components/views/Credentials/CredentialShare/index.tsx [hunk 1]
```diff
@@ -22,7 +23,15 @@ export const CredentialShare: React.FC<CredentialShareProps> = (props) => {
 
   return (
     <>
-      {props.children || <Link onClick={() => setModalOpen(true)}>Share</Link>}
+      {props.children || (
+        <UnstyledButton onClick={() => setModalOpen(true)}>
+          <HoverPathFill>
+            <Box p={1}>
+              <ShareIcon />
```
Comment (sruddy) [inline src/components/views/Credentials/CredentialShare/index.tsx:30]: @tobek I updated this button to match design I don't think we were using a button that just said `share` anywhere but wanted to double check that I didn't override something

File: src/components/views/Credentials/Credential.tsx [hunk 2]
```diff
@@ -1,301 +1,144 @@
...
   const expirationDate = vc.expirationDate && dateTimeFormat(new Date(vc.expirationDate));
-  const issuerFormatted = ellipsis("0x" + issuer.split("0x").pop(), 6, 4);
-  const issuerDomains = additionalVCData
-    ? additionalVCData.didListings.find((listing) => listing.did === issuer)?.domains
-    : [];
-  const subjectDomains = additionalVCData
-    ? additionalVCData.didListings.find((listing) => listing.did === vc.credentialSubject.id)?.domains
-    : [];
-
+  const expirationDateFormatted = expirationDate && `${expirationDate[0]} and ${expirationDate[1]}`;
```
Comment (tobek) [inline src/components/views/Credentials/Credential.tsx:34]: Minor, I think these should be "at" not "and"

File: src/components/views/Credentials/CredentialShare/index.tsx (hunk 1 above)
Comment (tobek) [reply src/components/views/Credentials/CredentialShare/index.tsx:30]: Not at all, that was just a placeholder until these updates, thanks for checkin

**Review by tobek — APPROVED** [review]
Splendid

**Pull request merged**